lib/logos.js

```diff
@@ -54,28 +54,56 @@ function getShieldsIcon({ name, color }) {
   }
 }
 
-function getSimpleIcon({ name, color }) {
+function brightness({ r, g, b }) {
+  return +((r * 299 + g * 587 + b * 114) / 255000).toFixed(2)
+}
+
+function hexToRgb(hex) {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
```

Comment:
I don't see this regex being used outside of this function.
Would it be better to keep it in here until its needed elsewhere?

Reply:
I think moving it outside is a performance optimisation. It gets declared once when we start the server process instead of redefining it on every function call